rpc/v8/subscriptions.go

```diff
@@ -757,13 +707,17 @@ type SubscriptionTransactionStatus struct {
 }
 
 // sendTxnStatus creates a response and sends it to the client
-func (h *Handler) sendTxnStatus(w jsonrpc.Conn, status SubscriptionTransactionStatus, id uint64) error {
+func sendTxnStatus(w jsonrpc.Conn, status SubscriptionTransactionStatus, id uint64) error {
+	return sendResponse("starknet_subscriptionTransactionsStatus", w, id, status)
+}
+
+func sendResponse(method string, w jsonrpc.Conn, id uint64, result any) error {
```

Comment:
Could we use generics here 👀 . Is it worth it?

Reply:
I think currently not worth it, because there are only 1 argument with type T and it's not constrained by any interface.